### ISSUE TITLE
Request: More settings options

### ISSUE THREAD

**Issue by Totemaster** [PrehistoricKingdom/feedback#466]
I just started the tutorial section, but I got stuck almost immediately on because my laptop doesn't have a "middle mouse button", and when I hold down the right and left mouse buttons then I can't move the view because it's all on one mousepad. Thus I am completely unable to rotate the camera.

I then exited the game, and tried altering the settings. However, although there were options for changing various other controls, the only one on the list that couldn't be edited was rotation.
Could you maybe add an option for changing it, please?

Thank you.

**Comment by shinryu1991** [comment]
Agree, I don't quite know why you're not able to bind any keys to rotating left or right. Feels physically unpleasant to have to hold the scroll wheel to be able to turn the camera either direction on a regular basis.
